### PR TITLE
Updating CONTRIBUTING guide for services info

### PR DESCRIPTION
**Pull request by rosinghal**
## Description 
Updating CONTRIBUTING guide for services info

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 NodeJS 14.x and NPM v8.x version
 
+# Services & ENV variables
+
+## Mandatory
+- [MongoDB](https://account.mongodb.com/account/login) - create an account and populate `DATABASE_URL` env variable.
+- [Twilio](https://www.twilio.com/) - create an account and populate `TWILIO_SID`, `TWILIO_API_KEY` and `TWILIO_NUMBER` env variables.
```
Comment (danishxtendify) [inline CONTRIBUTING.md:9]: @rosinghal its a TWILIO_API_KEY or Auth Token?

Comment (rosinghal) [reply CONTRIBUTING.md:9]: Auth token is the API key. Good catch.

Comment (rosinghal) [reply CONTRIBUTING.md:9]:
```suggestion
- [Twilio](https://www.twilio.com/) - create an account and populate `TWILIO_SID`, `TWILIO_API_KEY` (auth token) and `TWILIO_NUMBER` env variables.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
 
 NodeJS 14.x and NPM v8.x version
 
+# Services & ENV variables
+
+## Mandatory
+- [MongoDB](https://account.mongodb.com/account/login) - create an account and populate `DATABASE_URL` env variable.
```
Comment (rosinghal) [inline CONTRIBUTING.md:8]:
```suggestion
- [MongoDB](https://account.mongodb.com/account/login) - create an account and populate `DATABASE_URL` env variable. Follow [this](https://www.mongodb.com/docs/guides/cloud/connectionstring/) guide to get the database URL.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,6 +2,22 @@
...
+- SendGrid to send emails. Populate `SENDGRID_API_KEY` env variable.
+- Hubpost to sync contacts. Populate `HUBSPOT_CLIENT_ID` and `HUBSPOT_CLIENT_SECRET` env variables.
+- Mixpanel for adding events. Populate `MIXPANEL_TOKEN` env variable.
+- Sentry for bugs monitoring. Populate `SENTRY_DSN` env variable.
+- Stripe for taking payments. Populate `STRIPE_SECRET_KEY` and `STRIPE_PLAN_SUBSCRIPTION_LIST` env variables.
+
+
 # Clone & Setup
 
 Run `git clone git@gitlab.com:xtendify/founderphone/founderphone.git && cd founderphone`.
```
Comment (danishxtendify) [inline CONTRIBUTING.md:23]: @rosinghal it should be github URL, no?

Comment (danishxtendify) [inline CONTRIBUTING.md:23]: @rosinghal it should be github URL, no?

Comment (danishxtendify) [reply CONTRIBUTING.md:23]: Same for line number 35, I am not able to comment on line number 35.

Comment (rosinghal) [inline CONTRIBUTING.md:23]:
```suggestion
Run `git clone https://github.com/Xtendify/FounderPhone.git && cd FounderPhone`.
```

Comment (rosinghal) [reply CONTRIBUTING.md]: Updated, please check again

**Pull request closed without merging**